base/docs/helpdb.jl

```diff
@@ -3320,29 +3320,30 @@ Creates an iterable object for visiting each index of an AbstractArray `A` in an
 
 Example for a sparse 2-d array:
 
-    julia> A = sprand(2, 3, 0.5)
-    2x3 sparse matrix with 4 Float64 entries:
-        [1, 1]  =  0.598888
-        [1, 2]  =  0.0230247
-        [1, 3]  =  0.486499
-        [2, 3]  =  0.809041
+```jldoctest
+    julia> A = sparse([1, 1, 2], [1, 3, 1], [1, 2, -5])
```

Comment:
from the other `jldoctest` examples, it looks like it shouldn't have a leading indent on the input lines in the markdown version